src/middlewared/middlewared/plugins/network.py

```diff
@@ -389,7 +389,7 @@ def iface_extend(self, iface):
                 if lower_mtu and member_iface.mtu != lower_mtu and member_name in members_configured:
                     iface.delete_port(member_name)
                     members_configured.remove(member_name)
-                proc = await Popen(f'/sbin/ifconfig {member_name} {member_options}', shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, close_fds=True)
+                proc = await Popen(['/sbin/ifconfig', member_name, member_options], stdout=subprocess.PIPE, stderr=subprocess.PIPE, close_fds=True)
```

Comment:
It is not possible to set multiple options at the same time. Either the name member_option**s** is misleading or the following should work:

```
# ifconfig bridge create
bridge2
# python3.6
Python 3.6.4 (default, Mar 15 2018, 01:14:06) 
[GCC 4.2.1 Compatible FreeBSD Clang 4.0.0 (tags/RELEASE_400/final 297347)] on freebsd11
Type "help", "copyright", "credits" or "license" for more information.
>>> import subprocess
>>> subprocess.Popen(["ifconfig", "bridge2", "192.168.3.2/24 name bridge4"])                                                                                                                                                      
<subprocess.Popen object at 0x8007e19b0>
>>> ifconfig: 192.168.3.2/24 name bridge4: bad value (width invalid)
```

Here's the plain shell command executed on the host:
```
# ifconfig bridge2 192.168.3.2/24 name bridge4
bridge4
# ifconfig bridge4
bridge4: flags=8843<UP,BROADCAST,RUNNING,SIMPLEX,MULTICAST> metric 0 mtu 1500
        ether 02:cc:7d:c4:da:02
        inet 192.168.3.2 netmask 0xffffff00 broadcast 192.168.3.255 
        nd6 options=9<PERFORMNUD,IFDISABLED>
        groups: bridge 
        id 00:00:00:00:00:00 priority 32768 hellotime 2 fwddelay 15
        maxage 20 holdcnt 6 proto rstp maxaddr 2000 timeout 1200
        root id 00:00:00:00:00:00 priority 32768 ifcost 0 port 0
```

You could write:
```python
proc = await Popen(['/sbin/ifconfig', member_name] + member_options.split(), ...)